recipe/common.php

```diff
@@ -18,13 +18,21 @@
 set('clear_paths', []);         // Relative path from deploy_path
 set('clear_use_sudo', true);    // Using sudo in clean commands?
 
+
+/**
+ * Composer options
+ */
+env('composer_action', 'install');
+env('composer_options', '{{composer_action}} --verbose --prefer-dist --no-progress --no-interaction --no-dev --optimize-autoloader');
```

Comment:
@elfet I think it is ok. Not break BC and users can customize `composer_action` easily.
I thinnk in Deployer, `composer_options` will not include `composer_action`